graphics/macgui/mactextwindow.h

```diff
@@ -72,6 +72,7 @@ class MacTextWindow : public MacWindow {
 	int getTextSlant() { return _mactext->getTextSlant(); }
 	int getTextSlant(int start, int end) { return _mactext->getTextSlant(start, end); }
 	int getTextHeight() { return _mactext->getTextHeight(); }
+	int getLineHeight(int line) { return _mactext->getLineHeight(line); }
```

Comment:
This change must go into a separate commit. E.g. first, you add a commit `GRAPHICS: MACGUI: Adding getLineHeight() to MacTextWindow` and then follow by your `DIRECTOR: ...` commit. So, each commit stays compilable